templates/base.html

```diff
@@ -0,0 +1,60 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>UpTV</title>
+    <meta name="description" content="Never miss on your favorite shows!">
+
+    <!-- Common style sheets -->
+    <link rel="stylesheet" href="{% static 'base.css' %}">
+
+    <!-- Extra style sheets -->
+    {% block stylesheets %}{% endblock %}
+</head>
+<body>
+<nav>
+    <ul class="navbar">
+        <li class="tab">
+            <a href="{% url 'search_series' %}" id="active">
+                Search
```

Comment:
these texts are not centered anymore either... You can see the underline is longer on the right.

Reply:
Fixed :tada: